.github/workflows/util/execute_and_retry.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+execute_and_retry () {
+  retry_counter=0
+  max_retry=$1
+  while [ $retry_counter -lt $max_retry ]; do
+   deployment_failed=0
+   eval "$2" || deployment_failed=$?
+
+   if [ $deployment_failed -eq 1 ]; then
+     eval "$3"
+     retry_counter=$(($retry_counter+1))
+     sleep 5
+   else
+     break
+   fi
```

Comment:
Is this saying "If the command $2 doesn't fail, then run $3"? Can you clarify what the inputs to this function are?

Reply:
$1: Retry count
$2: Command to attempt
$3: Cleanup Command